ML-Frameworks/pytorch-aarch64/build-wheel.sh

```diff
@@ -55,6 +55,25 @@ UTILS=/utils
 COMMON_UTILS=/common_utils
 PYTORCH_FINAL_PACKAGE_DIR=/artifacts
 
+# ccache support; enable via env var or command line flag
+USE_CCACHE=${USE_CCACHE:-0}
+if [[ "$*" == *--use-ccache* ]]; then USE_CCACHE=1; fi
+export USE_CCACHE="${USE_CCACHE}"
+CCACHE_HOST_DIR=${CCACHE_HOST_DIR:-"${PWD}/.ccache"}
```

Comment:
Can we leave any of these unset for ccache to set, instead of having another layer of defaults

Reply:
I guess I was asking for the rationale for the defaults, and whether all were needed.

So `CCACHE_ROOT` needs to be consistent with the volume we mount, so a variable makes sense.

But e.g. `CCACHE_MAXSIZE`, could just be left unset if the user doesn't specify right? Then ccache would set it to its default.

I think `CCACHE_HOST_DIR` is subtle, are we intentionally avoiding using the host user's ~/.ccache (probably, because the compilers are different!)? That might be worth a comment.